keanu-project/src/test/java/io/improbable/keanu/vertices/dbl/DifferentiatorEfficiencyTest.java

```diff
@@ -0,0 +1,40 @@
+package io.improbable.keanu.vertices.dbl;
```

Comment:
Do you think it would be useful to wrap this as a Benchmark so we can monitor changes to this perf?

Reply:
This tests "efficiency", which just means it's not doing any more calls to autodiff than needed. I think the benchmarks should try quite a few more complex graphs that vary in length, depth, width, etc. The benchmarks *could* use this for inspiration but I don't think there's much benefit to coupling the benchmarks and this unit test.